sdk/communication/AzureCommunicationChat/README.md

```diff
@@ -113,7 +113,7 @@ Using the APIs, users can also send typing indicators when typing a message and
 
 #### Initialization
 
-To instantiate ChatClient you will need the CommunicationServices resource endpoint, a CommunicationUserCredential created from a User Access Token, and optional options to create the client with.
+To instantiate a ChatClient you will need the CommunicationServices resource endpoint, a CommunicationTokenCredential created from a User Access Token, and optional options to create the client with.
```

Comment:
Is this rename a breaking change that should be called out, or was the readme just wrong?

Reply:
Was already called in changelog - chat's readme is out of date